lib/Pipes/TaggedFunctionKind.cpp

```diff
@@ -50,6 +52,107 @@ using TaggedFK = TaggedFunctionKind;
 void TaggedFK::getInvalidations(const Context &Ctx,
                                 TargetsList &ToRemove,
                                 const GlobalTupleTreeDiff &Diff) const {
+  return;
+}
+
+using IsolatedFK = IsolatedFunctionKind;
+void IsolatedFK::getInvalidations(const Context &Ctx,
+                                  TargetsList &ToRemove,
+                                  const GlobalTupleTreeDiff &Diff) const {
```

Comment:
You're using `auto` way too much. Please change this habit now.

**Do not abuse `auto`**. Use `auto` only if:
    1. The line is very long and you can't figure any other way to make `check-conventions` happy.
    2. The type is obvious since it's expressedly
 stated on the same line (e.g., `auto *Lol = dyn_cast<CallInst>(...)`).
    3. The type is very ugly (typically, iterators).

Reply:
I believe I have used `auto` appropriately so far. For instance, you can infer that the following line returns a ContainerType of `FileContainer` kind:
```cpp
const auto &Container = Ctx.getReadOnlyContainer<FileContainer>(BCRR);
```
Whereas, e.g., here:
```cpp
auto AttrMatcher = PM::create<Binary>("/Functions/*/Attributes").value();
```
It is necessary to make `check-conventions` happy. For the same, if the type returns a long `std::optional`, I needed to opt for `auto`. All the other cases are continuously being reviewed in order not to abuse `auto`.